### ISSUE TITLE
Display all messages in Outlook 2013

### ISSUE THREAD

**Issue by MisterM74** [cmjnisse/sogo-ispconfig#38]
Hello

How to view all messages in Outlook 2013 that any dates.
Is it in the configuration Sogo?
Or outlook 2013 I have to set this bug?

Thanks for your help
Mz

**Comment by cmjnisse** [comment]
not sure what you mean.
do you use IMAP or activesync?

if you use activesync you properly need to set a higher timeout, see [SOGo - Microsoft Enterprise ActiveSync](https://github.com/inverse-inc/sogo/blob/master/Documentation/SOGoInstallationGuide.asciidoc#microsoft-enterprise-activesync), A LOT of new settings has been added to SOGo so you need to set them in the custom xml field.

**Comment by MisterM74** [comment]
Hello
Google translation
ActiveSync
What I mean, it displays the first posts of the week, this month.
But the messages of the past weeks or months, that it does not appear, it's more bother, if you understand my problem ...
When you talk about function to add, or are they?
Thank you, Michel

**Comment by cmjnisse** [comment]
Hmm, think i know what you mean.

Normal mail clients that can use ActiveSync by default only fetches the last week or two of emails, not sure about outlook, but there should be some settings in outlook to control this.

**Comment by MisterM74** [comment]
Indeed, when installing Office 2013 on the account, you must perform at least five operations:
1) show all messages "Offline"
2) Create a local account pop-type
3) Copy the messages in a folder pop account
4) Delete via webmail, hidden emails are always displayed, but "greyed out" mode.
5) copy the directory created (POP) to the account on your server.

From there, all your messages will appear.

Mz
